docs/topics/configmap-mission-intro.adoc

```diff
@@ -5,9 +5,14 @@ This mission shows you how to:
 * Use the configuration provided by the `ConfigMap` within an application.
 * Deploy changes to the `ConfigMap` configuration of running applications.
 
+.About Externalized Configuration
+It is important for an application's configuration to be externalized and separate from its code. This allows for the application's configuration to change as it moves through different environments while leaving the code unchanged. This also keeps sensitive or internal information out of your codebase and version control. Many languages and application servers provide environment variables to support externalizing an application's configuration.
+Kubernetes increases the complexity of this by adding pods, or groups of containers representing a deployment, and polyglot environments.
```

Comment:
Kubernetes increases the complexity of what?

Reply:
This sentence  used to say  "Many languages and application servers provide environment variables to support externalizing an application’s configuration. Microservices and Linux containers increase the complexity of this by adding pods, or groups of containers representing a deployment, and polyglot environments. ....
 Prior comment indicated was that wasn't accurate....    Kubernetes was added there.
What language would you like to see here?

Reply:
Line 9: separate -> separated.
Regarding line 10, I'd rather say something like the following:
Microservice architectures, where applications are split in independent services, add a layer of complexity to managing an application's configuration since, in this paradigm, configuration is not centralized anymore but ends up being as distributed and polyglot as the services that compose the application. This obviously creates a challenge of keeping all the configuration information in synch and accessible from all services.

Reply:
@lavernw1 please see my previous comment